src/js/ko-components/view-models/orifice-calculator.coffee

```diff
@@ -5,40 +5,64 @@
 class OPL.KoComponents.ViewModels.OrificeCalculator
   constructor: () ->
     @pipeID = ko.observableArray _.values OPL.OrificeCalculator.Config.Dictionaries.PipeID
-    @selectedPipeID = ko.observable OPL.OrificeCalculator.Config.Dictionaries.PipeID.one_nine_inch
-    @operatingPressure = ko.observable(0).extend {
-      required: { params: true, message: 'Please enter the operating pressure' }
-      digit: { params: true, message: 'Please enter an integer' }
-    }
+    @selectedPipeID = ko.observable OPL.OrificeCalculator.Config.Dictionaries.PipeID.oneNineInch
+    @operatingPressure = ko.observable(0).extend
+      required:
+        params: true
+        message: OPL.OrificeCalculator.Config.Dictionaries.Messages.operatingPressureError
+      digit:
+        params: true
+        message: OPL.OrificeCalculator.Config.Dictionaries.Messages.integerError
+
+    @operatingPressureWarningMessage = ko.pureComputed (->
+      if (@operatingPressure() > 200 and @operatingPressure() < 401)
+        return OPL.OrificeCalculator.Config.Dictionaries.Messages.operatingPressureWarningMayResult
+      else if (@operatingPressure() > 400)
+        return OPL.OrificeCalculator.Config.Dictionaries.Messages.operatingPressureWarningWillResult
+      else
+        return ''
```

Comment:
returning undefined instead of '' will make you have one less logical operation on the template. Check above comment (on template file)